src/components/UNSTABLE_Pagination/Pagination.stories.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file Unstable pagination stories.
+ * @copyright IBM Security 2019
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { UNSTABLE_Pagination, PageSelector } from '../..';
```

Comment:
Where did we land on the naming? What was your preference between `unstable__Pagination` and `UNSTABLE__Pagination`?

Reply:
Ah right, forgot to mention that in my comment last week -- 

I had trouble using `unstable` (lowercase) due to component conventions starting with an uppercase letter. 🤔 I was getting errors all over the place and didn't seem worth it to try to get around.
So do you mind if we keep the uppercase `UNSTABLE` for that reason? And maybe use two underscore instead of the one?

Reply:
Absolutely!

Reply:
This is a quick reminder to update the component name from `UNSTABLE_*` to `UNSTABLE__*`